src/components/SingleReview.jsx

```diff
@@ -1,31 +1,78 @@
 import { useParams } from 'react-router-dom';
 import { useEffect, useState } from 'react';
-import { getReviewById } from '../api';
+import { getCommentsByReviewId, getReviewById } from '../api';
+import RenderComments from './RenderComments';
 
 function SingleReview() {
 
     const [review, setReview] = useState({})
+    const [comments, setComments] = useState([])
     const { id } = useParams(); 
+    const [isLoading, setIsLoading] = useState(false)
+
 
     useEffect(() => {
+        setIsLoading(true)
         getReviewById(id).then((currReview) => {
             setReview({...currReview})
+            setIsLoading(false)
+        }).then(() => {
+            getCommentsByReviewId(id).then((currComments) => {
+                setComments([...currComments])
+            })
         })
     }, [])
 
-    console.log(review)
+
+    if (isLoading) {
+        return <p>Review Loading...</p>
+    }
+
+    if (comments.length === 0) {
+        return (
+            <>
+            <section className="review">
+                <h2>{review.title}</h2>
+                <img className="reviewImg"src={review.review_img_url} alt={review.title}/>
+                <p className="reviewBody">{review.review_body}</p>
+                <h3>Additional Information: </h3>
+                <ul className="reviewInfo">
+                    <li> Owner: {review.owner}</li>
```

Comment:
li tags could be something more appropriate are these list items, or are they different properties of a piece of data?